src/kvstore/RocksEngine.h

```diff
@@ -166,16 +174,19 @@ class RocksEngine : public KVEngine {
                                 std::unique_ptr<KVIterator>* iter) override;
 
   nebula::cpp2::ErrorCode prefix(const std::string& prefix,
-                                 std::unique_ptr<KVIterator>* iter) override;
+                                 std::unique_ptr<KVIterator>* iter,
```

Comment:
why don't use KVIterator* directly?

Reply:
Yes, it is bad to pass pointer of smart pointer. I don't know why it is coded this way. Maybe I should try to refactor it. But it do affect too many files.  Don't want do this in this pr.